clouddriver-kubernetes/src/main/groovy/com/netflix/spinnaker/clouddriver/kubernetes/model/KubernetesServerGroup.groovy

```diff
@@ -77,6 +79,10 @@ class KubernetesServerGroup implements ServerGroup, Serializable {
   }
 
   Boolean isDisabled() {
+
+    if (kind == "DaemonSet")
```

Comment:
According to the docs, you can: https://kubernetes.io/docs/concepts/workloads/controllers/daemonset/#communicating-with-daemon-pods

Looks like it works just like ReplicaSets/Deployments with a `.spec.selector`.